Snippets/TransactionalSession/TransactionalSession_1/AzureStorageConfig.cs

```diff
@@ -0,0 +1,68 @@
+﻿using NServiceBus;
+using NServiceBus.ObjectBuilder;
+using NServiceBus.TransactionalSession;
+using System.Threading.Tasks;
+using Microsoft.Azure.Cosmos;
+using NServiceBus.Persistence.AzureTable;
+
+namespace TransactionalSession_1
+{
+    public class AzureStorageConfig
+    {
+        public void Configure(EndpointConfiguration config)
+        {
+            #region enabling-transactional-session-azurestorage
+
+            config.UsePersistence<AzureTablePersistence>().EnableTransactionalSession();
+
+            #endregion
+        }
+
+        public async Task OpenDefault(IBuilder builder)
```

Comment:
We seem to be mixing `IServiceProvider` and `IBuilder` terminology even in the v1 version. I think we should align on one way